### PR TITLE
Update index.rst : adding svelte to regex

### PR DESCRIPTION
**Pull request by youassi**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Tickets       | fix doc : adding svelte to regex
| License       | MIT

### CODE REVIEW

File: src/Svelte/doc/index.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ code to your ``assets/app.js`` file:
     // assets/app.js
     import { registerSvelteControllerComponents } from '@symfony/ux-svelte';
 
-    registerSvelteControllerComponents(require.context('./svelte/controllers', true, /\\.(j|t)sx?$/));
+    registerSvelteControllerComponents(require.context('./svelte/controllers', true, /\.(ts|js|svelte)$/i));
```
Comment (1ed) [inline src/Svelte/doc/index.rst:63]:
```suggestion
    registerSvelteControllerComponents(require.context('./svelte/controllers', true, /\.(tsx?|jsx?|svelte)$/i));
```

Comment (weaverryan) [inline src/Svelte/doc/index.rst]: Yes, this is a mistake! It should actually match the recipe exactly: https://github.com/symfony/recipes/blob/main/symfony/ux-svelte/2.9/manifest.json

Can you update it to match?

Comment (youassi) [reply src/Svelte/doc/index.rst]: @weaverryan  done !

**Comment by weaverryan** [comment]
Thanks @youassi!

**Pull request merged**